src/wp-admin/includes/class-wp-posts-list-table.php

```diff
@@ -1100,7 +1100,8 @@ public function column_title( $post ) {
 		$pad = str_repeat( '&#8212; ', $this->current_level );
 		echo '<strong>';
 
-		$title = _draft_or_post_title();
+		$title = $post->post_title;
+		$title = empty( $title ) ? __( '(no title)' ) : strip_tags( $title, array( 'sub', 'sup' ) );
```

Comment:
```suggestion
		$title = empty( $title ) ? __( '(no title)' ) : strip_tags( $title );
```

@audrasjb What happen if we remove second argument `$allowable_tags` from it?

Reply:
@mukeshpanchal27 if we remove second argument `$allowable_tags`, the HTML tags are stripped by `strip_tags()` so superscript and subscript text are displayed as normal text.

Reply:
Oh crap. I didn't see that passing an array to `$allowable_tags` breaks on old PHP versions :(
Let's remove it completely, then.

Reply:
If you'd prefer to allow some tags you could use kses instead. For a small amount of text with a small number of allowed tags it's actually quite performant.